{
	"document_title": "CVE-2010-3495",
	"document_type": "SUSE CVE",
	"document_publisher": {
		"type": "Vendor",
		"contact_details": "security@suse.de",
		"issuing_authority": "SUSE Security Team"
	},
	"document_tracking": {
		"identification": {
			"id": "SUSE CVE-2010-3495"
		},
		"status": "Interim",
		"version": "1",
		"revision_history": {
			"revision": {
				"number": "3",
				"date": "2021-06-09T08:57:05Z",
				"description": "current"
			}
		},
		"initial_release_date": "2021-05-30T12:53:37Z",
		"current_release_date": "2021-06-09T08:57:05Z",
		"generator": {
			"engine": "cve-database/bin/generate-cvrf-cve.pl",
			"date": "2020-12-27T01:00:00Z"
		}
	},
	"document_notes": [
		{
			"text": "CVE-2010-3495",
			"title": "CVE",
			"type": "Summary"
		},
		{
			"text": "Race condition in ZEO/StorageServer.py in Zope Object Database (ZODB) before 3.10.0 allows remote attackers to cause a denial of service (daemon outage) by establishing and then immediately closing a TCP connection, leading to the accept function having an unexpected return value of None, an unexpected value of None for the address, or an ECONNABORTED, EAGAIN, or EWOULDBLOCK error, a related issue to CVE-2010-3492.",
			"title": "Mitre CVE Description",
			"type": "Description"
		},
		{
			"text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
			"title": "Terms of Use",
			"type": "Legal Disclaimer"
		}
	],
	"document_references": [
		{
			"type": "Self",
			"url": "https://lists.opensuse.org/opensuse-security-announce/2010-12/msg00006.html",
			"description": "E-Mail link for SUSE-SR:2010:024"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/support/security/rating/",
			"description": "SUSE Security Ratings"
		}
	],
	"product_tree": {},
	"vulnerability": [
		{
			"notes": [
				{
					"text": "Race condition in ZEO/StorageServer.py in Zope Object Database (ZODB) before 3.10.0 allows remote attackers to cause a denial of service (daemon outage) by establishing and then immediately closing a TCP connection, leading to the accept function having an unexpected return value of None, an unexpected value of None for the address, or an ECONNABORTED, EAGAIN, or EWOULDBLOCK error, a related issue to CVE-2010-3492.",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2010-3495",
			"threats": [
				{
					"type": "Impact",
					"description": "moderate"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {
					"base_score_v_2": "4.3",
					"vector_v_2": "AV:N/AC:M/Au:N/C:N/I:N/A:P"
				},
				"score_set_v_3": {}
			}
		}
	]
}
